apis/management.cattle.io/v3/catalog_types.go

```diff
@@ -25,6 +25,7 @@ type CatalogSpec struct {
 	CatalogKind string `json:"catalogKind,omitempty"`
 	Username    string `json:"username,omitempty"`
 	Password    string `json:"password,omitempty" norman:"type=password"`
+	HelmVersion string `json:"helmVersion,omitempty" norman:"noupdate"`
```

Comment:
shouldnt this be in status too?

Reply:
So status are things that are not set by the user, in this case we want the user to be able to set the helmversion at the catalog level on create, so it needs to live in the spec

Reply:
roger